README.md

```diff
@@ -1,147 +1,210 @@
-# C# ALGORITHMS [![Build Status](https://travis-ci.org/aalhour/C-Sharp-Algorithms.svg?branch=master)](https://travis-ci.org/aalhour/C-Sharp-Algorithms)
+```
```

Comment:
This file shouldn't be changed in your PR.

Reply:
ohhhh，i give it up😂